x-pack/legacy/plugins/epm/common/types.ts

```diff
@@ -108,28 +124,19 @@ export interface Dataset {
   type: string;
 }
 
-export interface RegistryPackage {
-  name: string;
-  version: string;
-  description: string;
-  readme?: string;
-  icon: string;
-  requirement: RequirementsByServiceName;
-  title?: string;
-  screenshots?: ScreenshotItem[];
-  assets: string[];
-  datasets: Dataset[];
+// some properties are optional in Registry responses but required in EPM
+// internal until we need them
+interface PackageAdditions {
+  title: string;
+  assets: AssetsGroupedByServiceByType;
```

Comment:
Change the shape of the value as we do in https://github.com/jfsiii/kibana/blob/575467c5eb6013b640f5f56589c3b809f7355bd7/x-pack/legacy/plugins/epm/server/packages/get.ts#L69

Reply:
Lets see if we can serve it in the future directly the way it is needed.

Reply:
I think this might be a case where it's ok for EPM to do extra work. Or if EPR adds this nested list, I don't think it should lose the existing flat list of contents.

Reply:
agree, it would be on top of that. But lets handle it on the EPM side for now and see if we can simplify things down the road.